source/virtualBuffers/MSHTML.py

```diff
@@ -76,15 +76,25 @@ def _normalizeControlField(self,attrs):
 		description=None
 		ariaDescribedBy=attrs.get('HTMLAttrib::aria-describedby')
 		if ariaDescribedBy:
-			try:
-				descNode=self.obj.rootNVDAObject.HTMLNode.document.getElementById(ariaDescribedBy)
-			except (COMError,NameError):
-				descNode=None
-			if descNode:
+			ariaDescribedByIds=ariaDescribedBy.split(" ")
+			for ariaDescribedById in ariaDescribedByIds:
+				descNode=None				
 				try:
-					description=self.obj.makeTextInfo(NVDAObjects.IAccessible.MSHTML.MSHTML(HTMLNode=descNode)).text
-				except:
+					descNode=self.obj.rootNVDAObject.HTMLNode.document.getElementById(ariaDescribedBy)
+				except (COMError,NameError):
+					descNode=None	
+				if descNode:

```

Comment:
Rather than if descNode: pass else: please change it to simply just if not descNode: